lib/resources.yml

```diff
@@ -63,11 +63,13 @@ js:
       - angular2: '*'
     React (latest):
       - react: '*'
+      - react-dom: '*'
 
   sourceFormat:
     options:
       - ES5
       - ES6
+      - JSX+ES6
```

Comment:
JSX in theory could be used with different things, but not without a lot of toggling and additional dependencies.
Its primary use is with React, and while there are things like a TSX transpiler built into TypeScript, and an angular-jsx library on npm, I'm not sure I'd be confident to implement those and get them working.

I can try to work out how to move JSX to its own question, and then we can use that as a hook for TypeScript / angular support later if necessary. Otherwise I would need to somehow nest the question under the React framework.